tensorflow_datasets/core/dataset_info_test.py

```diff
@@ -301,6 +302,13 @@ def test_updates_on_bucket_info(self):
     # These are dynamically computed, so will be updated.
     self.assertEqual(40, info.splits.total_num_examples)
     self.assertEqual(2, len(info.as_proto.schema.feature))
+    
+  def test_show_examples(self):
+    with testing.tmp_dir(self.get_temp_dir()) as tmp_dir:
+      builder = RandomShapedImageGenerator(data_dir=tmp_dir)
+    builder.download_and_prepare()
+    ds = builder.as_dataset(split='train')
```

Comment:
Maybe `split='test'`be faster

Reply:
Actually, its a dummy dataset with few random images. So, I think it will not vary much.